lib/exposure/exposure.js

```diff
@@ -187,6 +189,23 @@ export default class Exposure {
     }
 
     /**
+     * Initializes the reactive endpoint to retrieve the count of the data.
+     */
+    initCountPublication() {
+        const collection = this.collection;
+
+        genCountEndpoint(this.name, {
+            getCursor(session) {
+                return collection.find(session.filters, {}, this.userId);
```

Comment:
avoid using this in generated functions because you'll have to bind it, and binding the function uses additional memory